Ticket - 26691 Populate the itemfg.setupDate field when FG item is created and make sure utility does not inactivate new items.

sewa67 opened the pull request:
Please review this change and merge into development branch.
Files - util/setfgia.w, viewers/itemfg.w

sewa67 committed:
Ticket - 26691 Populate the itemfg.setupDate field when FG item is created and make sure utility does not inactivate new items.
Files - util/setfgia.w, viewers/itemfg.w

brad-advantzware reviewed (changes requested):
This appears to only be set when FG Items are created through the Viewer UI.  Need to also assign this value when items are created through Order/Job Entry (create an order or job with estimate that has a blank FG Item#)

sewa67 committed:
Ticket - 26691 Populate the itemfg.setupDate field when FG item is created and make sure utility does not inactivate new items.
Files - ce\b-estitm.w, ce\b-estitm1.w, ce\v-est.w, cec\b-estitm.w, cec\b-test.w, cec\v-est.w, est\b-estitm.w, est\fgadd2.p,
 est\NewFGFromEst.p, fg\ce-addfg.p, jc\fgadd.i, oe\d-oeitem.w, oe\fgadd2.p, oe\oelibupd.p, oe\oelibupl.p, oe\v-ord.w, oe\v-ordl.w,
 oe\v-ordpremier.w

Commit:
Merge branch 'develop' into feature/26691_Populate_the_itemfg_setupDate_field_when_FG_item_is_created_and_make_sure_utility_does_not_inactivate_new_items
